adhocracy-plus/templates/account/email/account_already_exists.en.email

```diff
@@ -0,0 +1,21 @@
+{% extends 'email_base.'|add:part_type %}
+{% load i18n %}
+
+{% block subject %}{% blocktranslate %}Account Already Exists{% endblocktranslate %}{% endblock %}
+
+{% block content %}
+<p>
+    {% blocktranslate %}You are receiving this e-mail because you or someone else tried to signup for an
+    account using e-mail address:{% endblocktranslate %}
+</p>
+<p>
+    {{ user.email }}
```

Comment:
This does not put my email into the mail, it should be `{{ email }}`

Reply:
@fuzzylogic2000 for me it only works with `user.email`. `email` only, shows the memory address